docs/contributing/developing.rst

```diff
@@ -331,3 +331,23 @@ To do this, you can run the following command to see the changes automatically a
 
 .. _Databases documentation: https://docs.djangoproject.com/en/stable/ref/databases/
 .. _DATABASES: https://docs.djangoproject.com/en/stable/ref/settings/#databases
+
+
+Automatically lint and code format on commits
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Pre-commit <https://pre-commit.com/>`_ is configured to automatically run code linting and formatting checks with every commit. You can run this manually inside of your virtual environment by:
+
+.. code-block:: console
+
+    $ pre-commit
+
+Alternatively having pre-commit setup on your local computer will also run the pre-commit hooks on every commit.
+Local installation can be done by one of the methods mentioned in the `pre-commit documentation <https://pre-commit.com/#installation>`_
+To install pre-commit into your git hooks run:
+
+.. code-block:: console
+
+    $ pre-commit install
+
+pre-commit should now run on every commit you make.
```

Comment:
I think we should only document this second approach, as this is the intended way to use pre-commit

```suggestion
`pre-commit <https://pre-commit.com/>`_ is configured to automatically run code linting and formatting checks with every commit. To install pre-commit into your git hooks run:

.. code-block:: console

    $ pre-commit install

pre-commit should now run on every commit you make.
```

Reply:
Good idea, more simplified too :)